app/src/main/java/org/mifos/mobilebanking/ui/fragments/LoanApplicationFragment.java

```diff
@@ -154,7 +156,7 @@ public void onCreate(Bundle savedInstanceState) {
 
     @Override
     public View onCreateView(LayoutInflater inflater, ViewGroup container,
-            Bundle savedInstanceState) {
+                             Bundle savedInstanceState) {
```

Comment:
Remove these kind of changes in the PR.

Reply:
Don't know what happened there... probably IDE stuff.